erpnext/selling/doctype/quotation/quotation.js

```diff
@@ -163,6 +164,15 @@ erpnext.selling.QuotationController = erpnext.selling.SellingController.extend({
 		};
 	},
 
+	contact_query: function(doc) {
+		return {
+			filters: {
+				link_doctype: frappe.dynamic_link.doctype,
+				link_name: doc.party_name
+			}
+		};
+	},
+
```

Comment:
`sales_common.js` line 41, a query for this Link field already exists (hence the filters that were visible earlier). Maybe check why that doesn't work and fix that query